internal/controllers/balance.go

```diff
@@ -77,3 +81,47 @@ type Balance struct {
 	// Balance is the total amount of $DIMO held at this time, across all chains.
 	Balance *big.Int `json:"balance" swaggertype:"number" example:"237277217092548851191"`
 }
+
+// GetHistoricalConversion godoc
+// @Description Calculate DIMO token earned fo a given week and popints
+// @Param        points query    int     true  "Number of points"
+// @Param        time   query    string  false  "Time in the week to calculate potential tokens earned based on the provided points (defaults to last week) (format 2006-01-02T15:04:05Z07:00)"
+// @Success      200    {object} HistoricalConversionResponse
+// @Router       /rewards/convert [get]
+func (r *RewardsController) GetHistoricalConversion(c *fiber.Ctx) error {
+	dateStr := c.Query("time")
+	var weekID int
+	if dateStr != "" {
+		weekTime, err := time.Parse(time.RFC3339, dateStr)
+		if err != nil {
+			return fiber.NewError(fiber.StatusBadRequest, fmt.Sprintf("invalid time format use %s", time.RFC3339))
```

Comment:
I think returning the Go "template time" is gonna be kinda confusing. Can we say "RFC-33339"?